noir-projects/aztec-nr/aztec/src/encrypted_logs/log_assembly_strategies/default_aes128/note.nr

```diff
@@ -50,7 +48,7 @@ where
     fields
 }
 
-pub fn compute_log<Note, let N: u32>(
+pub fn compute_note_log<Note, let N: u32>(
```

Comment:
Instead of hackily misusing the Note::get_id() to contain the log_type I now explicitly pass that info to the log functions.

Reply:
Makes sense. I'd add a comment to `compute_log` explaining that it can be either a private note log or a partial note private log, and that the encoding for both is prety much the same, so we just reuse the function.

Reply:
We will likely very soon split these two and have an encoding fn for private and one for partial. A partial note's `pack` fn currently includes the completion tag, which makes absolutely no sense.